src/main/java/mandel/fallingsand/Main.java

```diff
@@ -0,0 +1,22 @@
+package mandel.fallingsand;
+
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Sand sand = new Sand(50, 10);
+        Scanner input = new Scanner(System.in);
+        sand.randomSand(50);
+        while (true) {
+            System.out.println("Enter");
+            String enter = input.nextLine();
+            if (enter.isEmpty()) {
```

Comment:
Can you change it so that it goes until all the sand settles?

Reply:
It should settle all the way down each time the user presses enter? So before they press enter again, all the sand from the previous enter is already settled?

Reply:
settle all the way down, don't even bother having them press enter.

Reply:
Oh ok will try that

Reply:
Hi Professor Schwimmer,
I committed my most updated code, but it's not showing up on this page. It's in the repository and when I try to commit again it says there's nothing new to add, but it's not showing up here and it's not going through checkstyles. Are you able to see the code or is there something else I should do to try to re-push it?
Thank you

Reply:
You committed and then pushed?

Reply:
Yes